app/scripts/modules/core/src/application/ApplicationComponent.tsx

```diff
@@ -101,7 +98,7 @@ export class ApplicationComponent extends React.Component<IApplicationComponentP
 
     const Found = !this.props.app.notFound ? (
       <h2>
-        <Icon app={this.props.app} />
+        <ApplicationIcon app={this.props.app} />
```

Comment:
For non-cloud-provider `@Overridable` components, use them like normal react components